workorder.lua

```diff
@@ -464,9 +464,9 @@ local function create_orders(orders)
                     condition.has_tool_use = tmp
                 end
 
-                condition.anon_1 = -1
-                -- condition.anon_2 = ?
-                condition.anon_3 = -1
```

Comment:
Another option (albeit less explicit) is to set the `init-value` for both of these fields to `-1` in df-structures. That would make `df.manager_order_condition_item:new()` initialize them to -1. Not really necessary to change the script, IMO, but I'll add an init-value for `reaction_id` just to make other scripts a bit safer (`min_dimension` already has it).